spring-kafka-docs/src/main/antora/modules/ROOT/pages/testing.adoc

```diff
@@ -339,7 +339,7 @@ public class EmbeddedKafkaConditionTests {
 }
 ----
 
-A stand-alone (not Spring test context) broker will be created if the class annotated with `@EmbeddedBroker` is not also annotated (or meta annotated) with `ExtendedWith(SpringExtension.class)`.
+A standalone broker (outside the Spring's TestContext) will be created unless a class annotated `@EmbeddedKafka` is also annotated (or meta-annotated) with `ExtendWith(SpringExtension.class)`.
```

Comment:
the next line still uses "meta annotated". So changing above alone ends up with some inconsistency, thogh super minor.

Reply:
Fixed; thanks.